librz/util/skyline.c

```diff
@@ -11,6 +11,10 @@
 
 RZ_API bool rz_skyline_add(RzSkyline *skyline, RzInterval itv, void *user) {
 	rz_return_val_if_fail(skyline, false);
+	if (rz_itv_size(itv)) {
+		// Allow size 1 itv at UT64_MAX, block all other overflows
```

Comment:
@GustavoLCR why this? Does this block itv.addr == UT64_MAX-1, len = 2 as well?

Reply:
Judging from the unit tests you wrote, it seems that UT64_MAX-1, len=2 is possible indeed. Maybe it's just the comment wrong?